iUAEParents/MainEmulationViewController.mm

```diff
@@ -326,4 +384,250 @@ - (void)dealloc
     [super dealloc];
 }
 
+/******* START MCSESSION mithrendal ****/
+extern int mainMenu_servermode;
```

Comment:
Another minor thing: I think it would be nicer to move all this mcsession stuff into a new class, as much as that is possible.